project/Dependencies.scala

```diff
@@ -136,7 +135,6 @@ object Dependencies {
   // One reason to specify an override here is to avoid static-analysis security warnings.
   val transitiveDependencyOverrides = Seq(
     "commons-codec"                 % "commons-codec"         % "1.15",
-    "org.apache.httpcomponents"     % "httpclient"            % "4.5.13",
```

Comment:
this is a bonus fix unrelated to the `datarepo-client` update. We are already using `httpclient` version 4.5.13, so this override is extraneous. Removed.